enterprise/metrics/src/test/java/org/neo4j/metrics/output/CsvOutputTest.java

```diff
@@ -90,18 +88,18 @@ public void shouldHaveAbsoluteMetricsCsvPathBeAbsolute() throws Exception
     {
         // GIVEN
         File storeDir = directory.directory();
-        File file = File.createTempFile( "neo4j", "csvoutput" );
+        File outputFPath = new File( System.getProperty( "java.io.tmpdir" ), "output" );
```

Comment:
just curious, what was wrong with `createTempFile`? because that looks like more mature